src/main/java/com/puppycrawl/tools/checkstyle/site/PropertiesMacro.java

```diff
@@ -62,7 +63,10 @@ public class PropertiesMacro extends AbstractMacro {
     /** A newline with 20 spaces of indentation. */
     private static final String INDENT_LEVEL_20 = SiteUtil.getNewlineAndIndentSpaces(20);
 
-    /** The name of the current module being processed. */
+    /** The name of the token property being processed. */
+    private static final String TOKENS_PROPERTY = "tokens";
+
+    /** The file of the current module being processed. */
```

Comment:
I am new to review. Pardon me, if my questions sound silly, You changed the description from `name` to `file`. Was it not in sync with the functionality of the field?

Reply:
Changing comment from file to name will not effect, All checks are passing @relentless-pursuit